### PR TITLE
Fix new search query in code sample

### PR DESCRIPTION
**Pull request by alallema**
Fix a lot of mistake in code sample

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -357,7 +351,7 @@ getting_started_add_documents_md: |-
     var movies []map[string]interface{}
     json.Unmarshal(byteValue, &movies)
 
-    update, err := client.Index("movies").AddDocuments(movies)
+    _, err := client.Index("movies").AddDocuments(movies)
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:354]: In some samples you leave the variable even if it is not used in the same asmple 

For example: 

```
faceted_search_update_settings_1: |-
  response, err := client.Index("movies").UpdateAttributesForFaceting(&[]string{
    "director",
    "genres",
  })
```

How do you decide when you `_` the variable ?

Comment (alallema) [reply .code-samples.meilisearch.yaml:354]: You're right I think it better to leave the name everywhere.
Also, I realize that sometimes it's `result` and sometimes `response`.

Comment (alallema) [reply .code-samples.meilisearch.yaml:354]: Ah but this one it's the getting started. So I change it to be ready to use if you copy-paste the code

Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:354]: I think you're right!

**Review by bidoubiwa — APPROVED** [review]
LGTM

**Comment by alallema** [comment]
bors merge

**Pull request merged**